Snakefile

```diff
@@ -1060,6 +1060,9 @@ rule prepare_sector_network:
         network=RESDIR
         + "prenetworks/elec_s{simpl}_{clusters}_ec_l{ll}_{opts}_{sopts}_{planning_horizons}_{discountrate}_{demand}_presec.nc",
         costs="resources/" + RDIR + "costs_{planning_horizons}.csv",
+        # TODO revise the values which are currently very rough estimations
+        # &replace a temporary file with a more stable solution in future
+        cooling_costs="data/costs_cooling.csv",
```

Comment:
Ok for the temporary solution; ideally they shall be integrated in costs.csv but ok for now. Good to keep a todo about it

Reply:
Absolutely agree, and than you for opening the issue on that. Have added a link to the PR